orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForUpInstancesTask.groovy

```diff
@@ -219,4 +240,24 @@ class WaitForUpInstancesTask extends AbstractWaitingForInstancesTask {
 
     return deployment?.capacity as Map<String, Integer>
   }
+
+  public static class Splainer {
+    List<String> messages = new ArrayList<>()
+
+    def add(String message) {
+      messages.add(message)
```

Comment:
one concern is the relative timing information from many messages in a single session will be lost.
can do a string.format({time} - {message}) type thing here to alleviate that

Reply:
good point, I didn't consider timing at all for this